modules/fedimint-dummy-client/src/lib.rs

```diff
@@ -300,6 +300,7 @@ impl ClientModule for DummyClientModule {
                         // Since Done also happens for inputs we will fire too often, but that's ok
```

Comment:
Nit: I don't think this comment is super applicable anymore

Reply:
Removed :+1: